src/slave/containerizer/docker.hpp

```diff
@@ -257,8 +257,12 @@ class DockerContainerizerProcess
 
     static std::string name(const SlaveID& slaveId, const std::string& id)
     {
-      return DOCKER_NAME_PREFIX + slaveId.value() + DOCKER_NAME_SEPERATOR +
-        stringify(id);
+      std::string slaveIdstring = slaveId.value();
+      std::transform(slaveIdstring.begin(), slaveIdstring.end(),
```

Comment:
Here we force the Docker container name to be lower case to prevent bugs elsewhere.